### ISSUE TITLE
Making AFEB 100% web-based

### ISSUE THREAD

**Issue by vincentmorneau** [OraOpenSource/apex-frontend-boost#150]

**Comment by vincentmorneau** [comment]
For now instead of going full-Electron / native app, we can look into merging some of the commands.

Example instead of doing `afeb config <project>`, we could do `afeb <project>` which would open the configuration for a given project. From that interface, there would be a way to "Run" AFEB on the current project.

A side navigation containing the available projects could replace the `afeb list` command.

Basically the web interface would be the entry point to manage everything.

It would still require to use npm and the CLI to launch the web interface.

**Comment by martindsouza** [comment]
@vincentmorneau I'd hold off on this change then for now as we may have people who still want to run from CLI. Perhaps lets wait for feedback about the new UI / process before upgrading. If you agree let's tag this as "on hold"

**Comment by vincentmorneau** [comment]
100% agree, I was just laying off ideas for the future.

**Comment by vincentmorneau** [comment]
with https://github.com/vincentmorneau/json-local-configurator being the foundation of the new config UI, the current issue will most likely never be done. Closing.
